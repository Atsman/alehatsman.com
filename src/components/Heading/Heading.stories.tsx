import React from 'react'

import { storiesOf } from '@storybook/react'

import { Heading } from './Heading'

storiesOf('Heading', module)
  .add('default', () => (
    <Heading color="black">Hello Styled System</Heading>
  ))
