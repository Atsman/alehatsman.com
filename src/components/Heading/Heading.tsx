import React from 'react'
import styled from '@emotion/styled'
import { space, layout, color, typography } from 'styled-system'

export const Heading = styled.h1(
  {
    margin: 0,
    fontWeight: "bold"
  },
  space,
  layout,
  color,
  typography
)
