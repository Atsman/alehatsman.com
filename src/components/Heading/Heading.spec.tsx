import React from 'react'
import { render } from '@testing-library/react'
import { Heading } from './Heading'

describe('Heading', () => {
  it('should match snapshot', () => {
    expect(
      render(<Heading>This is heading content</Heading>).asFragment()
    ).toMatchSnapshot()
  })

  it('should render content', () => {
    const { getByText } = render(<Heading>Content</Heading>)
    expect(getByText('Content')).toBeInTheDocument()
  })
})
