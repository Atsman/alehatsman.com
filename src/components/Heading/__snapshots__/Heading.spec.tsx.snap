// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should match snapshot 1`] = `
<DocumentFragment>
  .emotion-0 {
  margin: 0;
  font-weight: bold;
}

<h1
    class="emotion-0 emotion-1"
  >
    This is heading content
  </h1>
</DocumentFragment>
`;
